src/app/components_ngrx/toolbar-panel/toolbar-panel.component.ts

```diff
@@ -391,7 +391,15 @@ export class ToolbarPanelComponent implements OnInit, AfterViewInit, OnDestroy {
         datasource: this.workItemTypeData,
         datamap: (witypes: WorkItemTypeUI[]) => {
           return {
-            queries: witypes.map(witype => {return {id: witype.id, value: witype.name, iconStyleClass: witype.icon}}),
+            queries: witypes.sort((a, b) => {
+              return (a.name > b.name ? 1 : 0)
```

Comment:
A semicolon is missing.

Reply:
A byte saved... hallelujah!!!

Semicolon in a single line return statement has no programmatic impact.
However, I'll update this commit to match with the linter expectations.